lib/SourceMapDevToolPlugin.js

```diff
@@ -102,12 +154,22 @@ class SourceMapDevToolPlugin {
 			new SourceMapDevToolModuleOptionsPlugin(options).apply(compilation);
 
 			compilation.hooks.afterOptimizeChunkAssets.tap(
-				{
-					name: "SourceMapDevToolPlugin",
-					context: true
-				},
+				//TODO need to fix type of Tap, currently { name: string; type: TapType; fn: Function; stage: number; context: boolean; }
```

Comment:
don't know is there any related issue

Reply:
This is fixed in tapable 2, which is used in the webpack next branch

Reply:
So your suggestion is to provide esdoc for `next` branch instead?

Reply:
@sokra ping

Reply:
Cast it to `TODO` here. This has already been fixed in the `next` branch.

``` js
tap(
  /** @type {TODO} */ ({
    name: "SourceMapDevToolPlugin",
    context: true
  }),
```